backend/setup.go

```diff
@@ -136,30 +155,82 @@ func SetupTracer(pluginID string, tracingOpts tracing.Opts) error {
 		// Initialize global tracer for plugin developer usage
 		tracing.InitDefaultTracer(otel.Tracer(pluginID))
 	}
-	Logger.Debug("Tracing", "enabled", tracingCfg.IsEnabled(), "propagation", tracingCfg.Propagation)
+
+	Logger.Debug(
+		"Tracing",
+		"enabled", tracingCfg.IsEnabled(),
+		"propagation", tracingCfg.propagation,
+		"samplerType", tracingCfg.sampler.SamplerType,
+		"samplerParam", tracingCfg.sampler.Param,
+		"samplerRemoteURL", tracingCfg.sampler.Remote.URL,
+		"samplerRemoteServiceName", tracingCfg.sampler.Remote.ServiceName,
+	)
 	return nil
 }
 
 // tracingConfig contains the configuration for OTEL tracing.
 type tracingConfig struct {
-	Address     string
-	Propagation string
+	address     string
+	propagation string
```

Comment:
Nit. Breaking change, but guess should be fine. Should write something about it though in the upcoming release

Reply:
`type tracingConfig struct` was unexported and only returned by `getTracingConfig` (which was also unexported), so it wasn't available for plugin developers thus it should be fine.

I have also unexported `IsEnabled` in this commit: https://github.com/grafana/grafana-plugin-sdk-go/pull/781/commits/a859f787bcaea83b34011641d2cde1fe54672ead because it doesn't make sense to export it either

Reply:
Sorry my bad 👍